packages/embeds/js/src/features/bubble/components/Bubble.tsx

```diff
@@ -148,7 +150,8 @@ export const Bubble = (props: BubbleProps) => {
       <div
         part="bot"
         style={{
-          height: 'calc(100% - 80px)',
+          height: props.height ?? 'calc(100% - 80px)',
+          width: props.width ?? '100%',
```

Comment:
That is not good.

You can try it out in the `packages/react` package, run `stories:dev`

Reply:
If you can post screenshots 🙏